packages/Search/src/DTK_DistributedSearchTree_decl.hpp

```diff
@@ -97,8 +97,8 @@ class DistributedSearchTree
 
   private:
     Teuchos::RCP<Teuchos::Comm<int> const> _comm;
-    BVH<DeviceType> _local_tree;
-    BVH<DeviceType> _distributed_tree;
+    BVH<DeviceType> _top_tree;    // replicated
+    BVH<DeviceType> _bottom_tree; // local
```

Comment:
If I didn't know what `bottom_tree` and `top_tree` were, there is no way I would understand what they are.